test/rpc/test_rpc_misc.py

```diff
@@ -200,6 +201,59 @@ def test_get_events_wrong_blockid_type():
     assert resp["error"]["code"] == PredefinedRpcErrorCode.INVALID_PARAMS.value
 
 
+@pytest.mark.usefixtures("run_devnet_in_background")
+@pytest.mark.parametrize(
+    "run_devnet_in_background",
+    [PREDEPLOY_ACCOUNT_CLI_ARGS],
+    indirect=True,
+)
+def test_get_events_continuation_token():
+    """
+    Test RPC get_events returning continuation token.
+    """
+    deploy_info = deploy(EVENTS_CONTRACT_PATH)
+    for i in range(3):
+        invoke(
+            calls=[(deploy_info["address"], "increase_balance", [i])],
+            account_address=PREDEPLOYED_ACCOUNT_ADDRESS,
+            private_key=PREDEPLOYED_ACCOUNT_PRIVATE_KEY,
+        )
+    resp = rpc_call(
+        "starknet_getEvents",
+        params=create_get_events_rpc(create_block_from_0(3))["params"],
```

Comment:
I see we are using the `create_get_events_rpc` function only to extract the `params` from it (also present in some old tests). This looks like a problem, and should be dealt with. If you see a simple solution that won't require several hours of refactoring, you can do it in this PR, otherwise I will create a new GitHub issue. But I need your feedback, please.

Reply:
If it does require extensive refactoring we could at least define a separate method that extracts this params and use it here. So it will be easier to replace eventually.